internal/runners/activate/activation.go

```diff
@@ -67,9 +67,9 @@ func (r *Activate) activateAndWait(proj *project.Project, venv *virtualenvironme
 
 	analytics.Event(analytics.CatActivationFlow, "before-subshell")
 
-	fail := <-r.subshell.Failures()
-	if fail != nil {
-		return locale.WrapError(fail, "error_in_active_subshell", "Failure encountered in active subshell")
+	err = <-r.subshell.Errors()
+	if err != nil {
+		return locale.WrapError(err, "error_in_active_subshell", "Failure encountered in active subshell")
```

Comment:
This is the only place where the "failures" -> "error" change is used in the State Tool.  As I renamed the function, I can't have missed anything.  And therefore, I am 99.99% sure that I am not accidentally wrapping an error in a failure. :)